.github/workflows/docker-build.yml

```diff
@@ -4,8 +4,13 @@ on:
   push:
     branches:
       - "main"
+      - "release/*"
```

Comment:
currently, we're using v0.X.X, but I'm okay with switching to release

can we do tags as well?